rules/config

```diff
@@ -161,3 +160,6 @@ SONIC_ENABLE_IMAGE_SIGNATURE ?= n
 
 # ENABLE_SYNCHRONOUS_MODE - enable synchronous mode between orchagent and syncd
 ENABLE_SYNCHRONOUS_MODE = n
+
+# Default SSH per user login limit (unlimited)
+DEFAULT_USER_LOGIN_LIMIT = 0
```

Comment:
Is this tested and working? According to the [man page](https://man7.org/linux/man-pages/man5/limits.conf.5.html), it appears that `-1` is the numerical representation of unlimited:

> All items support the values -1, unlimited or infinity indicating no limit, except for priority and nice.